src/api.js

```diff
@@ -79,23 +81,36 @@ const populate = (data) => {
     });
   });
   getLikes();
+  const games = document.querySelectorAll('div[id=game-container]');
+  counter(games);
 };
 
 const populateLikes = (data) => {
   data.forEach((element) => {
     const item = document.getElementById(element.item_id);
-    item.innerHTML = `Likes: ${element.likes}`;
+    if (item != null) {
+      item.innerHTML = `Likes: ${element.likes}`;
+    }
   });
 };
 
+const counter = (games) => {
+  let counter = 0;
+  games.forEach(() => {
+    counter += 1;
+  });
+  count.innerHTML = `Displaying ${counter} Games!`;
+  return counter;
+};
+
```

Comment:
I am pretty sure that `querySelectorAll` returns an array of elements. Don't you think It would be much DRY and clean to use `games.length`, since it will return the length of the array. To see how it works click [here](https://www.tutorialspoint.com/javascript/array_length.htm).